docs-source/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -0,0 +1,98 @@
+as similar as possible to the Pod that will later be generated for the Administration Server. This guarantees that the operator is
+analyzing the same WebLogic domain configuration that WebLogic Server instances will use.
+
+Introspection ensures that:
+1. The operator is aware of domain topology from the WebLogic domain configuration, including servers, clusters, network access points, listen addresses, and other configurations.
+2. The operator can generate configuration overrides to adjust the WebLogic domain configuration to match the Kubernetes environment, such as modifying listen addresses.
+3. For Model in Image, the operator can generate the WebLogic domain home, including the final domain configuration.
+4. For Domain in PV and Domain in Image, the operator can use any customer-provided [configuration overrides]({{<relref "/userguide/managing-domains/configoverrides/_index.md">}}) along with the operator-generated overrides to generate the final configuration overrides.
+
+#### When introspection occurs automatically
+
+Introspection automatically occurs when:
+1. The operator is starting a WebLogic Server instance when there are currently no other servers running. This occurs when the operator first starts servers for a domain or when starting servers following a full domain shutdown.
+2. For Model in Image, the operator determines that at least one WebLogic Server instance that is currently running must be shut down and restarted. This could be a rolling of one or more clusters, the shut down and restart of one or more WebLogic Server instances, or a combination.
+
+#### Initiating introspection
+
+Sometimes, such as for the [use cases](#introspection-use-cases) described below, it is desirable to explicitly initiate introspection. To initiate introspection, change the value of your Domain `introspectVersion` field.
+
+Set `introspectVersion` to a new value.
+
+```
+  kind: Domain
+  metadata:
+    name: domain1
+  spec:
+    introspectVersion: "2"
+    ...
+```
+
+As with `restartVersion`, the `introspectVersion` field has no required format; however, we recommend using a value likely to be unique such as a continually increasing number or a timestamp.
+
+#### Failed introspection
+
+When introspection fails, the operator will not start any WebLogic Server instances. If this is not the initial introspection and there are already WebLogic Server instances running, then a failed introspection will leave the existing WebLogic Server instances running without making any changes to the operational state of the domain.
+
+The introspection will be periodically retried and then will eventually timeout with the Domain `status` indicating the processing failed. To recover from a failed state, correct the underlying problem and update the `introspectVersion`.
+
```

Comment:
I think it'd be helpful to provide a debugging section here: minimally pointers to, or cut-and-pastes from, the model-in-image and config-overrides debugging sections.

Reply:
Also, just like in the introduction, consider working in a mention that the job is named `DOMAIN_UID-introspect-domain-job` and that it runs in the same namespace as the Domain.